src/commands/kv/namespace/create.rs

```diff
@@ -21,7 +27,22 @@ pub fn create(target: &Target, user: GlobalUser, title: &str) -> Result<(), fail
     });
 
     match response {
-        Ok(success) => message::success(&format!("Success: {:#?}", success.result)),
+        Ok(success) => {
+            message::success(&format!("Success: {:#?}", success.result));
+            match env {
+                Some(env) => message::success(&format!(
+                    "Add the following to your TOML under [env.{}]:",
```

Comment:
can we say `wrangler.toml` here instead of TOML to be consistent with other messages across the codebase?